cmd/s3.go

```diff
@@ -247,6 +249,9 @@ func (c *config) s3Delete() error {
 
 	response, err := c.cloud.S3.Deletes3(request, nil)
 	if err != nil {
+		if errors.Is(err, context.DeadlineExceeded) {
```

Comment:
Another approach would be to simply put this into background and tell the user that the deletion will take a while to complete

Reply:
So you want to keep the server connection open? Not sure if there are other timeouts involved which will cancel the request prematurely. 🤔